shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -277,14 +315,52 @@ - (void)replaceRange:(UITextRange*)range withText:(NSString*)text {
 }
 
 - (BOOL)shouldChangeTextInRange:(UITextRange*)range replacementText:(NSString*)text {
-  if (self.returnKeyType == UIReturnKeyDone && [text isEqualToString:@"\n"]) {
-    [self resignFirstResponder];
-    [self removeFromSuperview];
```

Comment:
Are these not still needed in the done case?

Reply:
I'm not 100% sure.  My experimentation with an actual app didn't show any problem without them, and my assumption is that when the Flutter side says "unfocus" that those same methods are being invoked in some other execution path.  But I'm not positive.

Reply:
FYI - @cbracken and I looked at Japanese and "done" style actions, as well as "next" style actions appear to work as desired with these lines removed.